src/Channels/WritableBuffer.cs

```diff
@@ -158,13 +159,27 @@ public void Write(Span<byte> source)
                 CommitBytes(writable);
             }
         }
+        /// <summary>
+        /// Writes a structure of type T to a buffer of bytes.
+        /// </summary>
+        public void WriteCpuEndian<[Primitive]T>(T value) where T : struct
```

Comment:
@davidfowl  I've removed this method from the PR; frankly, an undefined endian write method has no utility in an IO pipeline